fbchat/graphql.py

```diff
@@ -44,7 +44,7 @@ def get_customization_info(thread):
         rtn['nicknames'] = {}
         for k in info['participant_customizations']:
             rtn['nicknames'][k['participant_id']] = k.get('nickname')
-    else:
+    elif info['participant_customizations']:
```

Comment:
Maybe it's not really *that* important here, but in general, I think we should be as prepared as possible  ;)